composer.json

```
@@ -28,8 +28,7 @@
         "php": ">=5.4.0",
         "ext-dom": "*",
         "ext-mbstring": "*",
-        "phenx/php-font-lib": "0.5.*",
-        "phenx/php-svg-lib": "0.3.*"
+        "phenx/php-font-lib": "0.5.*"
```

Use newly released version instead:

```suggestion
        "phenx/php-font-lib": "0.5.*",
        "phenx/php-svg-lib": "^0.3"
```